src/gui/aboutdialog.cpp

```diff
@@ -19,23 +19,33 @@
 ***********************************************************************************************************************/
 
 #include "aboutdialog.h"
+#include "version.h"
```

Comment:
You need to change this line to:
#include "../core/version.h"

Reply:
fixed